src/atomic.rs

```diff
@@ -801,6 +801,8 @@ impl<'scope, T> Clone for Ptr<'scope, T> {
     }
 }
 
+impl<'scope, T> Copy for Ptr<'scope, T> {}
```

Comment:
I'm curious as well, actually... :/

Both `Copy` and `Clone` could be derived here. If any of those are derived, then I get errors like:

```
rror[E0507]: cannot move out of borrowed content
   --> src/registry.rs:139:38
    |
139 |                     .compare_and_set(self.curr, succ, AcqRel, self.scope)
    |                                      ^^^^ cannot move out of borrowed content
```